### ISSUE TITLE
Ability to preconfigure reminders for ALL events on a calendar

### ISSUE THREAD

**Issue by Porrumentzio** [nextcloud/server#25936]
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are interested into the same feature.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments. 

**Is your feature request related to a problem? Please describe.**
It happens, specially with the non-writable calendars like Birthdays, that you're not reminded of events; but you would like to be **reminded for all events**, even for those you will create, a week before (thus you can search a gift for your contact's birthday)

**Describe the solution you'd like**
It would be great if users could **preconfigure reminders**, and maybe other characteristics of events like recurrency, for all past and future events on a calendar
Summing up, to be able to configure some things by calendar, which will affect all its events

This way, users could benefit more from Birthday calendar (it's not doable to set reminders manually for each event) and also configure some characteristics for all events (recurrency, reminders...)
